### ISSUE TITLE
python: Installer needs to put Scripts dir into path 

### ISSUE THREAD

**Issue by rwarren** [ScoopInstaller/Main#377]
The python installer currently does not put the `Scripts` dir into the path. This means that installed scripts (or even `pip.exe` itself) aren't in the path.

To fix this, the following path needs to be added to the `%PATH%`:

`%HOMEPATH%\scoop\apps\python\current\Scripts`

**Comment by rwarren** [comment]
For `.py` scripts to work, the `py.exe` launcher (see [here](https://docs.python.org/3/using/windows.html#launcher)) also needs to be installed, since this is what recognizes script shebangs properly. I'm not sure how this would link to the required python.exe shim, but it seems like this is configurable (see [PEP-397](https://www.python.org/dev/peps/pep-0397/#configuration-file), the PEP that defines the `py.exe` implementation).

I'd love to do this myself, but my powershell-fu is non-existent, and I don't understand what the scoop installer json is doing (and not doing).

@r15ch13 It _seems_ like the launcher (and path modification?) is being explicitly excluded. Why is this? I ask you because of blame history on this seemingly relevant line:

https://github.com/ScoopInstaller/Main/blame/master/bucket/python.json#L19

**Comment by linsui** [comment]
Scripts has been added into path, see
https://github.com/ScoopInstaller/Main/blame/master/bucket/python.json#L53

**Comment by rwarren** [comment]
@linsui You are right. My apologies... I had something weird going on with my env var inheritance (due to explorer.exe respawns).

The real thing driving my issue creation is the lack of shebang support. Shebang support needs the `py.exe` launcher, per my second comment. I'll rename the issue accordingly.

**Comment by rwarren** [comment]
Fixing this will also likely require modifications to the default application to use for `.py` files.

**Comment by rasa** [comment]
`.py` is added to `PATHEXT` [here](https://github.com/ScoopInstaller/Main/blob/b5f34f3d5ab3fe3848259ec35f3126881b125e6e/bucket/python.json#L27).

**Comment by uranusjr** [comment]
Any objections about `py.exe`? It is a very recommended component, and I see no point excluding it intentionally.
